render/candlestick.go

```
@@ -0,0 +1,502 @@
+		termWidth = getTerminalWidth()
+	}
+
+	yaWidth := yAxisLabelWidth(maxHigh)
+	leftMargin := 1
+	chartAreaWidth := termWidth - leftMargin - yaWidth
+	if chartAreaWidth < 10 {
+		chartAreaWidth = 80 - leftMargin - yaWidth
+	}
+
+	numCandles := len(candles)
+	candleWidth := 3
```

golangci-lint (2.12.2)

[error] ineffectual assignment to candleWidth

(ineffassign)